### PR TITLE
[dart] Bump commit

### PR DESCRIPTION
**Pull request by chapulina**
I don't know if this is the correct thing to do. I'm trying to sync the macOS binaries with the latest Ubuntu ones to fix https://github.com/ignitionrobotics/ign-physics/pull/262/.

### CODE REVIEW

**Comment by chapulina** [comment]
build bottle

File: Formula/dartsim@6.10.0.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 class DartsimAT6100 < Formula
   desc "Dynamic Animation and Robotics Toolkit (openrobotics port)"
   homepage "https://dartsim.github.io/"
-  # osrc custom nightly built from commit fdde7e7894ebc36bae8811f7a63e5b1c899bb4af
-  url "https://github.com/azeey/dart/archive/1673b0be51fb370023df7490dc49706b590d8f72.tar.gz"
-  version "6.10.0~20200916~1673b0be51fb370023df7490dc49706b590d8f72"
-  sha256 "ec2e833d3225ac3f4365cc6d8b2f5511170a47d140ff43dcc7d63a50fbb6bfd5"
+  # OSRF's fork
+  url "https://github.com/ignition-forks/dart/archive/71c7cfb97e8b583aa1ada639b6d1cf9a1835c1da.tar.gz"
```
Comment (scpeters) [inline Formula/dartsim@6.10.0.rb:5]: I believe the debs are built from https://github.com/ignition-forks/dart/commit/175596b511adb55d831c0c3401ef50e2cb96a6f3

Comment (scpeters) [reply Formula/dartsim@6.10.0.rb:5]: I'll update this

Comment (scpeters) [reply Formula/dartsim@6.10.0.rb]: f151eb2

**Comment by scpeters** [comment]
build bottle

File: Formula/dartsim@6.10.0.rb (hunk 1 above)
Comment (chapulina) [reply Formula/dartsim@6.10.0.rb]: Thanks. The extra commit was just adding the changelog, so I think the end result won't be different.

Comment (scpeters) [reply Formula/dartsim@6.10.0.rb]: agreed, but this way it's easier to match up the debian and homebrew versions

**Pull request merged**